packages/server/test/test-server.ts

```
@@ -124,163 +124,162 @@ if ((globalThis as any)['createUWS']) {
+      }
+    },
   };
+};
+serverImplMap['fastify'] = async function createFastifyTestServer() {
+  let adapter: ServerAdapter<
+    {
+      req: FastifyRequest;
+      res: ServerResponse;
+      reply: FastifyReply;
+    },
+    ServerAdapterBaseObject<{}>
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)